apps/daffio/src/app/docs/guide-docs/components/table-of-contents/table-of-contents.component.html

```diff
@@ -0,0 +1,3 @@
+<div *ngFor="let entry of doc.tableOfContents.json" class="table-of-contents__item">
+	<a class="table-of-contents__item--level{{entry.lvl}}" routerLink="/{{doc.id}}" [fragment]="entry.slug">{{ entry.content }}</a>
```

Comment:
Can the `routerLink` be a `./` or is there any way to keep the path on the current route to avoid the dependency on the entire doc?

Reply:
I tried `./` and it seems to still work.

Reply:
ok nice. I can change address the `Can the input here just be the toc[]?` comment above then

Reply:
I'll wait till after @xelaint is done with design changes

Reply:
done w/ UI updates

Reply:
changed